common/src/main/webapp/sass/explore.scss

```diff
@@ -370,6 +370,10 @@ body {
   flex-direction: column;
 }
 
+.target-asterism-table {
+  max-height: 250px;
```

Comment:
This is inside a flex grid, maybe we can specify it using flex grow/shrink/basis?

Reply:
I guess a fixed max-height is fine. I just thought that we may get something similar if we make the aladin part grow faster than the table. Not sure when the overflow will be activated